solutions/devsprint-cecilia-saraiva-2/FinanceAppSnapshotTests/ActivityViewCellSnapshotTests.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  ActivityViewCellSnapshotTests.swift
+//  FinanceAppTests
+//
+//  Created by Raline Maria da Silva on 24/08/22.
+//
+
+import XCTest
+import SnapshotTesting
+@testable import FinanceApp
+
+class AcitivityCellViewSnapshotTests: XCTestCase {
+    
+    let size1: CGSize = .init(width: 320, height: 120)
+    let size2: CGSize = .init(width: 420, height: 120)
+    
+    override func setUp() {
+        SnapshotTesting.isRecording = false
+    }
+    
+    func testActivityCell() {
+        let activityCell = ActivityCellView()
+        activityCell.backgroundColor = .white
+        
+        assertSnapshot(matching: activityCell, as: .image(size: size1))
+        assertSnapshot(matching: activityCell, as: .image(size: size2))
```

Comment:
[sugestao] que tal colocar um nome pra cada teste através do parâmetro `named`? Dai poderíamos ter nomes um pouco mais significativos que `testActivityCell.1` e `testActivityCell.2`. 
```suggestion
        assertSnapshot(matching: activityCell, as: .image(size: size1), named: "ActivityCellView-320x120")
        assertSnapshot(matching: activityCell, as: .image(size: size2), named: "ActivityCellView-420x120")
```

Reply:
feito